lua/csharpls_extended/utils.lua

```diff
@@ -78,4 +78,20 @@ U.jump_to_location = function(location, bufnr)
     vim.api.nvim_win_set_cursor(0, { location.range.start.line + 1, location.range.start.character })
 end
 
+U.client_request_sync = function (client, a, b, c, d)
```

Comment:
Can the param be any other name? I think a, b, c ,d are bad..

Reply:
Resolved!